invenio_webhooks/models.py

```diff
@@ -174,10 +174,8 @@ def status(self, event):
         """Return a tuple with current processing status code and message."""
         result = AsyncResult(str(event.id))
         return (
-            self.CELERY_STATES_TO_HTTP.get(result.state),
-            result.info.get('message')
-            if result.state == states.PENDING and result.info
```

Comment:
We could add `states.STARTED` to the condition. But we should use the `event.response` anytime the task is finished or failed.